go.mod

```diff
@@ -1,4 +1,4 @@
-module iot-service/iot_service
+module github.com/NUS-EVCHARGE/iot-service
```

Comment:
I changing back to remote instead of local module. Let me know if has any concern. Open to feedback @sweiyang @gitkh2891